bears/vcs/git/GitCommitBear.py

```diff
@@ -305,25 +315,24 @@ def check_issue_reference(self, body,
             return
 
         host = self.get_host_from_remotes()
-        if host not in self.SUPPORTED_HOST_KEYWORD_REGEX:
+        if (host not in self.SUPPORTED_HOST_KEYWORD_REGEX or
+                host not in self.ISSUE_INFO):
             return
 
+        if body_close_issue_full_url:
+            self.issueType = 'full issue'
+        else:
+            self.issueType = 'issue'
+
         if body_close_issue_on_last_line:
             if body:
                 body = body.splitlines()[-1]
-            result_message = ('Body of HEAD commit does not contain any {} '
-                              'reference in the last line.')
-        else:
-            result_message = ('Body of HEAD commit does not contain any {} '
-                              'reference.')
-
-        if body_close_issue_full_url:
-            result_info = 'full issue'
-            issue_ref_regex = (
-                r'https?://{}\S+/issues/(\S+)'.format(re.escape(host)))
+            result_message = ('Body of HEAD commit does not contain any '
```

Comment:
if you keep `{} '` at the end of this line, it wont be modified, making your diff much smaller.

Also you dont need to change either of the `result_message = ...` statements.

You can just add below

```py
    result_message = result_message.format(self.issue_type)
```